gcp-connector-util build fails

<$ go get github.com/google/cloud-print-connector/...>

results in...

<# github.com/google/cloud-print-connector/gcp-connector-util go/src/github.com/google/cloud-print-connector/gcp-connector-util/init.go:417:26: multiple-value uuid.NewV4() in single-value context>

Am I missing something?

Confirmed I receive the same issue:
```
root@host:~# go get github.com/google/cloud-print-connector/...
# github.com/google/cloud-print-connector/gcp-connector-util
go/src/github.com/google/cloud-print-connector/gcp-connector-util/init.go:417:26: multiple-value uuid.NewV4() in single-value context
root@host:~# 
root@host:~# go version
go version go1.9.2 linux/amd64
```

--- a/gcp-connector-util/init.go
+++ b/gcp-connector-util/init.go
@@ -414,7 +414,11 @@ func initConfigFile(context *cli.Context) error {
 		if context.IsSet("proxy-name") {
 			proxyName = context.String("proxy-name")
 		} else {
-			proxyName = uuid.NewV4().String()
+			v4, err := uuid.NewV4()
+			if err != nil {
+				return err
+			}
+			proxyName = v4.String()
 		}
 
 		var userClient *http.Client

